include/generic_datatype.h~

```diff
@@ -0,0 +1,27 @@
+
+namespace smt {
```

Comment:
plz remove emacs tmp files.
`git rm include/generic_datatype.h~`

The way to not add them is to use `git add -u` instead of `git add -A`. This will only `add` existing files to the commit, and will ignore new ones. Then we will have another problem -- you will forget to add new files. But...